### ISSUE TITLE
About lines in NBV.cpp

### ISSUE THREAD

**Issue by inhwanlee** [sunwaylive/quality-driven-poisson-guided-autoscanning#8]
Hello,

I tried to run your code and found out that there was "assertion error" and the program stopped running at following line in NBV.cpp
view_grid_points->vert.push_back(t);//[index] = t;
It seems like the assertion error is from "component_ocf.h" line 620, in includeLib/vcglib/vcg/simplex/vertex folder.
It says following comment.
    // You should never ever try to copy a vertex that has OCF stuff.
		// use ImportData function.

But "view_grid_points->vert.push_back(t);//[index] = t;" this line tries to copy t (CVertex type) into view_grid_points->vert[index]. 
How did you make this code ran without problem?

Thank you.
Sincerely,
Inhwan Lee

**Comment by inhwanlee** [comment]
Ok. I tried in release_debug mode and the memory of CVertex became 184 from 200.
and there was no more error.
Does pure debug mode not work with this project?

**Comment by wsh312** [comment]
Yes, as far as I remember, there were some bugs in the debugging mode.
I apologize for our mistakes!

Best,
Shihao

inhwanlee <notifications@github.com> 于2018年7月12日周四 下午8:52写道：
